### ISSUE TITLE
Question: I am trying to auto_annotate images using google collab but getting this error.

### ISSUE THREAD

**Issue by Lucifer7535** [AlvaroCavalcante/auto_annotate#69]
**THE ERROR:**

`2023-04-13 13:39:21.365592: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-04-13 13:39:22.331926: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/lib64-nvidia
2023-04-13 13:39:22.332041: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/lib64-nvidia
2023-04-13 13:39:22.332060: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
Loading model into memory...
2023-04-13 13:39:25.709694: W tensorflow/core/common_runtime/gpu/gpu_bfc_allocator.cc:42] Overriding orig_value setting because the TF_FORCE_GPU_ALLOW_GROWTH environment variable is set. Original config value was 0.
Loading label map...
Found 1036 images to annotate.
  0% 0/1036 [00:00<?, ?it/s]'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
'_UserObject' object is not callable
  2% 20/1036 [00:00<00:05, 199.26it/s]'_UserObject' object is not callable`

Is this the problem with collab env or some other problem?

**Comment by Lucifer7535** [comment]
.
